components/trezor_bridge/resources/BUILD.gn

```diff
@@ -0,0 +1,39 @@
+import("//brave/components/common/typescript.gni")
+
+transpile_web_ui("trezor_bridge_ui") {
+  entry_points = [
+    [
+      "trezor_bridge",
+      rebase_path("trezor_bridge.ts"),
+    ],
+    [
+      "trezor/trezor",
+      rebase_path("trezor/iframe.ts"),
+    ],
+  ]
+
+  resource_name = "trezor_bridge"
+}
+
+pack_web_resources("generated_resources") {
+  resource_name = "trezor_bridge"
+  output_dir = "$root_gen_dir/brave/components/trezor_bridge/resources"
+  deps = [
+    ":data",
+    ":trezor_bridge_ui",
+  ]
+}
+
+node_data_dir = "//brave/node_modules/trezor-connect/data"
+
+copy("data") {
```

Comment:
This target should probably be moved to `trezor_bridge/BUILD.gn` as that's where the message handler that uses the resources is.

Reply:
done